Utilities/bin_patch.cpp

```diff
@@ -559,6 +546,7 @@ static std::basic_string<u32> apply_modification(const patch_engine::patch_info&
 		case patch_type::le32:
 		{
 			*reinterpret_cast<le_t<u32, 1>*>(ptr) = static_cast<u32>(p.value.long_value);
+			resval = offset;
```

Comment:
Who uses le32 for instructions? It seems very inconvenient.

Reply:
It's not about conveniency, it's about not introducing bugs which can be discovered by the user with some effort.

Reply:
Changed anyways.